src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -711,13 +758,15 @@ Result TimestampOrderingTransactionManager::CommitTransaction(
 
```

Comment:
May be it's better that we add some comments about all the _COMPILER_MEMORY_FENCE_ in the commit and abort method. Removing these fences can cause fatal race condition bugs...